src/storage/database.rs

```diff
@@ -91,22 +110,69 @@ mod tests {
                     content: "Hello world!".bytes().collect(),
                     compression: None,
                 },
-                backend.get("dir/foo.txt")?
+                backend.get("dir/foo.txt", std::usize::MAX)?
             );
 
             // Test that other files are not returned
             assert!(backend
-                .get("dir/bar.txt")
+                .get("dir/bar.txt", std::usize::MAX)
                 .unwrap_err()
                 .downcast_ref::<PathNotFoundError>()
                 .is_some());
             assert!(backend
-                .get("foo.txt")
+                .get("foo.txt", std::usize::MAX)
```

Comment:
The usages of `usize::MAX` seem strange to me, it seems like testing with a sane-ish number is better and more realistic than this, which is essentially not having a limit at all

Reply:
These tests are not testing the size limit, so there is no point in setting a lower limit.